rita_client/src/exit_manager/mod.rs

```diff
@@ -777,4 +779,20 @@ pub async fn exit_manager_tick() {
             }
         }
     }
+
+    trace!("Setting up ipv6 for slaac");
+    let rita_settings = settings::get_rita_client();
```

Comment:
this needs a block comment explaining what it's doing.

Reply:
fixed